commons/src/main/java/org/eclipse/kapua/commons/configuration/KapuaConfigurableServiceBase.java

```diff
@@ -80,6 +90,11 @@ public Map<String, Object> getConfigValues(KapuaId scopeId) throws KapuaExceptio
 
     @Override
     public void setConfigValues(KapuaId scopeId, KapuaId parentId, Map<String, Object> values) throws KapuaException {
+        // Argument Validation
+        ArgumentValidator.notNull(scopeId, "scopeId");
+        ArgumentValidator.notNull(parentId, "parentId");
+        ArgumentValidator.notEmptyOrNull(values.values(), "values");
```

Comment:
Here we should check also for `values` notNull

Reply:
Changed the checks to
```java
ArgumentValidator.notNull(scopeId, "scopeId");
for (Object value : values.values()) {
  ArgumentValidator.notNull(value, "values");
}
```